.ci.yaml

```diff
@@ -59,7 +59,9 @@ platform_properties:
           {"name":"flutter_cocoapods","path":"cocoapods"},
           {"name":"old_osx_sdk","path":"osx_sdk"},
```

Comment:
I'm not sure what `old_osx_sdk` should be renamed to given https://github.com/flutter/flutter/pull/99586 since the names were different between the engine and framework.